addon_service/common/known_imps.py

```diff
@@ -87,6 +88,9 @@ class AddonImpNumbers(enum.Enum):
     GOOGLE_DRIVE = 1005
     FIGSHARE = 1007
     ONEDRIVE = 1008
+    DATAVERSE = 1010
+
+
```

Comment:
Looks like this is causing problems with pre-commit; you should probably delete these two lines.